src/Skybrud.Umbraco.Redirects/Services/RedirectsService.cs

```diff
@@ -227,47 +244,9 @@ public class RedirectsService : IRedirectsService {
         /// <param name="url">The URL of the redirect.</param>
         /// <param name="queryString">The query string of the redirect.</param>
         /// <returns>An instance of <see cref="Redirect"/>, or <c>null</c> if not found.</returns>
+        [Obsolete("Use GetRedirectByPathAndQuery")]
```

Comment:
Since the package is still only available as an alpha release, I'm not so concerned about legacy support. So ideally this method should be removed entirely - both in the interface and in this class.